src/middlewared/middlewared/alembic/versions/22.12/2022-12-14_00-45_nfs_protocols.py

```diff
@@ -0,0 +1,45 @@
+"""empty message
```

Comment:
Let's put a message here

Reply:
Does "Replace v4 with protocols in nfs service configuration." seem sufficient?